client/post-editor/media-modal/detail/detail-title.jsx

```diff
@@ -69,10 +69,6 @@ export default React.createClass( {
 				title: this.state.title
 			} );
 		}
-
-		if ( this.isMounted() ) {
-			this.resetTitle();
```

Comment:
@aduth was the original intent here to make sure we used the item.title over state.title in getTitleValue()? Are there any cases you can think of that may break if we remove this?